.env.example

```diff
@@ -7,6 +7,9 @@ WP_ENV=development
 WP_HOME=http://example.com
 WP_SITEURL=${WP_HOME}/wp
 
+# Optional variables
```

Comment:
Can you move this up under the rest of the DB vars?

Also maybe we should treat `DB_HOST` as an optional comment too. Its default is `localhost`.